package.json

```diff
@@ -12,18 +12,20 @@
   ],
   "main": "./base",
   "scripts": {
-    "prebuild": "yarn run build:clean",
-    "build:clean": "rimraf build rules customRules base.js react.js",
+    "prebuild": "yarn run clean",
+    "clean:build": "rimraf build rules customRules '*.js'",
+    "clean": "yarn run clean:build",
```

Comment:
Hm, I think this might have just become stale and I didn't notice/update it. Thanks!